task/jq/0.1/jq.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: jq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.56.1"
+    tekton.dev/categories: Developer Tools
+    tekton.dev/tags: jq
+    tekton.dev/displayName: "JQ"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    Task built for using the JQ utility in pipelines.
+  workspaces:
+    - name: source
+      description: A workspace that contains the file which needs to be altered. It can take the form of a persistent volume (PV and PVC) or configmap.
+      optional: true
```

Comment:
isn't this workspace mandatory in order to run this task or can it take in a remote file as well?

Reply:
No, it isn't. This scenario (not using the workspace) is covered by tests.

What do you mean by remote file? Something like an URI where a file is located? If that's the case JQ Task does not cover it. It would be a good enhancement for a new version.

Reply:
@vinamra28 news?